addons/revolutionaries/CfgVehicles_O_Tanoan.hpp

```diff
@@ -145,6 +145,103 @@ class TACU_Revolutionaries_U_O_Tanoan_Autorifleman: TACU_Revolutionaries_U_O_Tan
     editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
 };
 
+class TACU_Revolutionaries_U_O_Tanoan_FigureHeadUA: TACU_Revolutionaries_U_O_Tanoan_Rifleman01 {
+    displayName = "Figure Head (Unarmed)";
+    uniformClass = "U_I_C_Soldier_Para_2_F";
+    linkedItems[] = {DEFAULT_ITEMS, "H_Beret_blk"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS, "", "H_Beret_blk"};
+    backpack = "";
+    weapons[] = {"Throw", "Put"};
+    respawnWeapons[] = {"Throw", "Put"};
+    magazines[] = {""};
+    respawnMagazines[] = {""};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
```

Comment:
```suggestion
    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
    headgearList[] = {
        "H_Beret_blk", 1
    };
```
CBA_facewearList as well if you don't want to use a balaclava.